src/nvim/memory.c

```diff
@@ -286,10 +286,11 @@ size_t memcnt(const void *data, char c, size_t len)
   return cnt;
 }
 
-/// The xstpcpy() function shall copy the string pointed to by src (including
+#ifndef HAVE_STPCPY
+/// The stpcpy() function shall copy the string pointed to by src (including
 /// the terminating NUL character) into the array pointed to by dst.
 ///
-/// The xstpcpy() function shall return a pointer to the terminating NUL
+/// The stpcpy() function shall return a pointer to the terminating NUL
 /// character copied into the dst buffer. This is the only difference with
 /// strcpy(), which returns dst.
```

Comment:
> xstpcpy is different than stpcpy.

I do remember that despite misgivings about the exact implementation when we did it earlier, we did try to follow the standard. How is it different?